wire-ios-request-strategy/Sources/Message Sending/MessageSenderTests.swift

```diff
@@ -75,11 +74,10 @@ final class MessageSenderTests: MessagingTestBase {
             .withApiVersionResolving(to: nil)
             .arrange()
 
-        // when
-        let result = await messageSender.sendMessage(message: message)
-
         // then
-        assertFailure(result: result, expectedFailure: MessageSendError.unresolvedApiVersion)
+        await assertItThrows(error: MessageSendError.unresolvedApiVersion) {
+            try await messageSender.sendMessage(message: message)
+        }
     }
 
     func testThatWhenSendingMessageSucceeds_thenReturnSuccessResult() async throws {
```

Comment:
```suggestion
    func testThatWhenSendingMessageSucceeds_thenDoesNotThrowError() async throws {
```

Reply:
Re-named all the tests in https://github.com/wireapp/wire-ios/pull/658